scale-testing.mk

```diff
@@ -0,0 +1,45 @@
+# run the following from the project root:
+# make -f scale-testing.mk <make rule>
+
+# AWS variables for cloud hosted k8s testing
+AWS_REGION ?= us-east-2
+EKS_K8S_VERSION ?= 1.30
+
+# directories for cloud hosted k8s infrastructure for running tests
+# root directory for all integration tests
+TF_EKS_SRC_DIR ?= $(INTEGRATION_TEST_ROOT)/infra/scale-testing/eks-cluster
+TF_EKS_STATE_DIR ?= $(TF_EKS_SRC_DIR)/state
+TF_DEPLOY_SRC_DIR ?= $(INTEGRATION_TEST_ROOT)/infra/scale-testing/deployments
+TF_DEPLOY_STATE_DIR ?= $(TF_DEPLOY_SRC_DIR)/state
+
+include ./Makefile
+
+.PHONY: create-eks
+create-eks: set-vault-license ## Create a new EKS cluster
+	@mkdir -p $(TF_EKS_STATE_DIR)
+ifeq ($(VAULT_ENTERPRISE), true)
+    ## ensure that the license is *not* emitted to the console
+	@echo "vault_license = \"$(_VAULT_LICENSE)\"" > $(TF_EKS_STATE_DIR)/license.auto.tfvars
+endif
```

Comment:
Do we still need `set-vault-license` on the eks target at all? Looks like we'll want to move the `@echo "vault_license ...` block down into `deploy-workload` as well?

Reply:
Good catch! We don't need `set-vault-license` on that target anymore.  I will update and move the license block. Thank you 🙏